### PR TITLE
delete-voucher

### PR DESCRIPTION
**Pull request by gingerbreadpup**

### CODE REVIEW

**Review by charlesstrange2 — COMMENTED** [review]
as a WIP, this looks like the right direction :+1:

File: src/main.js [hunk 1]
```diff
@@ -125,7 +126,7 @@ var vm = new Vue({
  * @param err
  *   Logout reason.
  */
-EventBus.$on('NetMgr.logout', (err) => {
+EventBus.$on('NetMgr.logout', function(err) {
```
Comment (katjam) [inline src/main.js:129]: :) fat arrow police! :policeman: :oncoming_police_car:

File: src/sass/app.scss [hunk 2]
```diff
@@ -938,6 +938,15 @@ h1.expandable.queue {
     &:nth-child(1) {
       width: 50%;
       text-align: left;
+      .icon {
+        display: inline;
+        font-size: 0.6em;
+        padding-left: 0.5em;
+        color: $arc_dark_gray;
+        a:hover {
```
Comment (katjam) [inline src/sass/app.scss:946]: Does this need a focus style too?

File: src/store.js [hunk 3]
```diff
@@ -266,6 +266,24 @@ store.addVoucherCode = function (voucherCode, success, failure) {
...
+ * informs the server we'v delted a voucher.
+ */
+store.delVoucher = function (voucherCode, success, failure)
+{
+    let transition = this.transitionVouchers('reject', [ voucherCode ], success, failure);
+
+    transition.then(function() {
+        let voucher = this.trader.vouchers[len - 1];
+        if(voucher) {
+            voucher.online = NetMgr.online;
```
Comment (katjam) [inline src/store.js:279]: Is this a flag to say whether the change happened while on or offline to check when trader loaded next time?

**Review by katjam — APPROVED** [review]
Nothing to stop you merging in. :yin_yang:

**Pull request merged**